features/web-audio.yml.dist

```diff
@@ -0,0 +1,732 @@
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "70"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioContext.getOutputTimestamp
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "58"
+  #   chrome_android: "58"
+  #   edge: "79"
+  #   firefox: "70"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioContext.baseLatency
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "64"
+  #   chrome_android: "64"
+  #   edge: "79"
+  #   firefox: "76"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.port
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "66"
+  #   chrome_android: "66"
+  #   edge: "79"
+  #   firefox: "76"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioParamMap
+  - api.AudioWorklet
+  - api.AudioWorkletGlobalScope
+  - api.AudioWorkletGlobalScope.currentFrame
+  - api.AudioWorkletGlobalScope.currentTime
+  - api.AudioWorkletGlobalScope.registerProcessor
+  - api.AudioWorkletGlobalScope.sampleRate
+  - api.AudioWorkletNode
+  - api.AudioWorkletNode.AudioWorkletNode
+  - api.AudioWorkletNode.parameters
+  - api.AudioWorkletNode.port
+  - api.AudioWorkletNode.processorerror_event
+  - api.BaseAudioContext.audioWorklet
+
+  # baseline: false
+  # support:
+  #   chrome: "41"
+  #   chrome_android: "41"
+  #   edge: "14"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.OfflineAudioContext.suspend
+
+  # baseline: false
+  # support:
+  #   chrome: "52"
+  #   chrome_android: "52"
+  #   edge: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioListener.forwardX
+  - api.AudioListener.forwardY
+  - api.AudioListener.forwardZ
+  - api.AudioListener.positionX
+  - api.AudioListener.positionY
+  - api.AudioListener.positionZ
+  - api.AudioListener.upX
+  - api.AudioListener.upY
+  - api.AudioListener.upZ
+
+  # baseline: false
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioParam.cancelAndHoldAtTime
+
+  # baseline: false
+  # support:
+  #   chrome: "68"
+  #   chrome_android: "68"
+  #   edge: "79"
+  #   safari: "14"
+  #   safari_ios: "14"
+  - api.AudioParam.automationRate
+
+  # baseline: false
+  # support:
+  #   chrome: "102"
+  #   chrome_android: "102"
+  #   edge: "102"
+  #   firefox: "70"
+  #   firefox_android: "79"
+  - api.AudioContext.outputLatency
```

Comment:
This is a FIXME in WebKit:
https://github.com/WebKit/WebKit/blob/312f06335038c0c43733c76223f9a53806a002c1/Source/WebCore/Modules/webaudio/AudioContext.idl#L38-L39